backend/cmd/main.go

```diff
@@ -39,8 +39,13 @@ func main() {
 		log.Fatalf("Error getting client: %v", err)
 	}
 
+	nativeClient, err := utils.GetNativeClient()
+	if err != nil {
+		log.Fatalf("Error getting native client: %v", err)
+	}
+
 	// create a type that satisfies the `api.ServerInterface`, which contains an implementation of every operation from the generated code
-	server := handlers.NewServer(oClient)
+	server := handlers.NewServer(oClient, nativeClient)
```

Comment:
```suggestion
	server := handlers.NewServer(oClient, nativeClient, liqoNamespace)
```

And you can get the namespace as argument as follows:

```
liqoNamespace := flag.String("liqo-namespace", "liqo", "The namespace where liqo is installed")
```

before `flag.Parse()`